sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-import-csv.service.ts

```diff
@@ -55,7 +55,8 @@ export class EducationProgramOfferingImportCSVService {
     return csvModels.map((csvModel) => ({
       offeringName: csvModel.offeringName,
       yearOfStudy: csvModel.yearOfStudy,
-      showYearOfStudy: csvModel.showYearOfStudy === YesNoOptions.Yes,
+      // showYearOfStudy will not be used anymore
+      showYearOfStudy: true,
```

Comment:
Even being part of the CSV should we keep parsing this?

Reply:
Ok. Removed that.